src/config/gobjects.rs

```diff
@@ -1,4 +1,3 @@
-use std::ascii::AsciiExt;
```

Comment:
It used.

Reply:
Uhh, where? :)  The compiler told me it was unused, and it built fine without it.

Reply:
Never mind, I see it now.  It's only required on stable; nightly doesn't need it.  I've added it back.